SupportYourLocals.Data/XMLData.cs

```diff
@@ -0,0 +1,164 @@
+﻿using MapControl;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace SupportYourLocals.Data
+{
+    public class XMLData : IDataStorage
+    {
+        private string filePath = @"./LocalSellersData.xml";
+        private string fileName = "LocalSellersData.xml";
+        XDocument doc = new XDocument();
+        XmlDocument document = new XmlDocument();
```

Comment:
Use the "const" keyword for variables that will never change (path, name) and "readonly" for variables that will only be set in the constructor and never again (doc, document)